base_user_role/i18n/am.po

```diff
@@ -0,0 +1,207 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * base_user_role
+# 
+# Translators:
+# OCA Transbot <transbot@odoo-community.org>, 2016
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 9.0c\n"
```

Comment:
I'm fairly certain this won't be an issue. Something something, automation is amazing